app/api/activitylog/activitylogParser.js

```diff
@@ -1,245 +1,180 @@
-    'Deleted multiple entities',
-    methods.delete
-  ),
-  'DELETE/api/documents': entitiesDELETE,
-  'POST/api/attachments/upload': generatePlainDescriptionBeautifier(
-    'Uploaded attachment',
-    methods.create
-  ),
-  'POST/api/attachments/rename': attachmentsRenamePOST,
-  'DELETE/api/attachments/delete': generateDeleteBeautifier('attachment', 'attachmentId'),
-  'POST/api/templates': generateCreateUpdateBeautifier('template', '_id', 'name'),
-  'POST/api/templates/setasdefault': templatesAsDefaultPOST,
-  'DELETE/api/templates': generateDeleteBeautifier('template', '_id'),
-  'POST/api/thesauris': generateCreateUpdateBeautifier('thesaurus', '_id', 'name'),
-  'DELETE/api/thesauris': generateDeleteBeautifier('thesaurus', '_id'),
-  'POST/api/relationtypes': generateCreateUpdateBeautifier('relation type', '_id', 'name'),
-  'DELETE/api/relationtypes': generateDeleteBeautifier('relation type', '_id'),
-  'POST/api/translations': translationsPOST,
-  'POST/api/translations/languages': translationsLanguagesPOST,
-  'DELETE/api/translations/languages': translationsLanguagesDELETE,
-  'POST/api/translations/setasdeafult': translationsAsDefaultPOST,
-  'POST/api/pages': generateCreateUpdateBeautifier('page', 'sharedId', 'title'),
-  'DELETE/api/pages': generateDeleteBeautifier('page', 'sharedId'),
-  'POST/api/settings': generatePlainDescriptionBeautifier('Updated settings'),
-  'POST/api/relationships/bulk': generatePlainDescriptionBeautifier('Updated relationships'),
-  'POST/api/references': generateCreateUpdateBeautifier('relationship', '_id'),
-  'DELETE/api/references': generateDeleteBeautifier('relationship', '_id'),
-  'POST/api/upload': generatePlainDescriptionBeautifier('Uploaded document', methods.create),
-  'POST/api/reupload': generatePlainDescriptionBeautifier('Re-uploaded document', methods.update),
-  'POST/api/customisation/upload': generatePlainDescriptionBeautifier(
-    'Uploaded custom file',
-    methods.create
-  ),
-  'DELETE/api/customisation/upload': generateDeleteBeautifier('custom file', '_id'),
-  'POST/api/import': generatePlainDescriptionBeautifier(
-    'Imported entities from file',
-    methods.create
-  ),
-  'POST/api/public': generatePlainDescriptionBeautifier(
-    'Created entity coming from a public form',
-    methods.create
-  ),
-  'POST/api/remotepublic': generatePlainDescriptionBeautifier(
-    'Submitted entity to a remote instance',
-    methods.create
-  ),
-  'POST/api/users/new': usersNewPOST,
-  'POST/api/semantic-search': semanticSearchPOST,
-  'DELETE/api/semantic-search': generateDeleteBeautifier('semantic search', 'searchId'),
-  'POST/api/semantic-search/stop': generateSemanticSearchUpdateBeautifier(
-    'Stopped semantic search'
-  ),
-  'POST/api/semantic-search/resume': generateSemanticSearchUpdateBeautifier(
-    'Resumed semantic search'
-  ),
-  MIGRATE: migrationLog,
+import * as helpers from 'api/activitylog/helpers';
```

Comment:
@mfacar this file was basically rewritten, I think it's a good opportunity to make it TS. You think that's relatively quick to accomplish?